tests/interfaces-network/task.yaml

```diff
@@ -0,0 +1,85 @@
+summary: |
```

Comment:
The summary should really be a summary. Something like "Ensure network interface works."

The content below is nice, though, and should go under:
```
details: |
    The network interface ...
```